README.rst

```diff
@@ -46,6 +46,56 @@ corner lets you edit the following information:
 
 * Access control: to save changes, you need to grant the IDE permission to write
   to your GitHub repo or, if you started with a D-Wave example, create a fork.
-* Environmental variables: the IDE automatically sets your SAPI token (token used
+* Environmental variables such as your SAPI token (token used
   to authenticate client sessions when you connect to remote solvers such as D-Wave
   quantum computers and hybrid solvers).
+
+Configuration
+-------------
+
+Some more advanced configurations include the following:
+
+Repository Requirements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create a YAML file to open a repository with your required packages
+loaded as follows::
+
+  tasks:
+   - init: pip install -r requirements.txt
+
+User Dockerfile
+~~~~~~~~~~~~~~~
+
+The IDE uses a Docker image, based on python:3.7-slim, available within
+the IDE as ``dwavesys/leapide:latest``. To build on top of it, your Dockerfile
+should look like this::
```

Comment:
Also, the dockerfile has to be [referenced from the `.gitpod.yml`](https://www.gitpod.io/docs/config-docker/).